sematic/ui/packages/common/src/pages/RunDetails/PipelineSection.tsx

```diff
@@ -43,29 +47,37 @@ const StyledPipelineTitle = styled(PipelineTitle)`
 `;
 
 const PipelineSection = () => {
+    const { rootRun } = useRootRunContext() as RemoveUndefined<ExtractContextType<typeof RootRunContext>>;
+
+    const { error, avgRuntime, successRate, totalCount }
+        = useBasicMetrics({ runId: rootRun.id, rootFunctionPath: rootRun.function_path });
+
     return (
         <>
             <TopSection>
                 <Headline>Pipeline</Headline>
-                <StyledPipelineTitle>MNIST PyTorch Example</StyledPipelineTitle>
+                <StyledPipelineTitle>{rootRun.name}</StyledPipelineTitle>
                 <ImportPath>
-          sematic.examples.mnist.pipeline.with.very.long.path
+                    {rootRun.function_path}
                 </ImportPath>
                 <StyledVertButton />
             </TopSection>
             <BottomSection>
-                <Box>
-                    <Typograph variant="bigBold">257</Typograph>
-                    <Typograph variant="body1">runs</Typograph>
-                </Box>
-                <Box>
-                    <Typograph variant="bigBold">74 min.</Typograph>
-                    <Typograph variant="body1">avg. runtime</Typograph>
-                </Box>
-                <Box>
-                    <Typograph variant="bigBold">62%</Typograph>
-                    <Typograph variant="body1">success</Typograph>
-                </Box>
+                {error && <Alert severity="error">{error.message}</Alert>}
+                {!error && <>
+                    <Box>
+                        <Typograph variant="bigBold">{totalCount}</Typograph>
+                        <Typograph variant="body1">runs</Typograph>
+                    </Box>
+                    <Box>
+                        <Typograph variant="bigBold">{`${avgRuntime}`}</Typograph>
```

Comment:
What happens if this becomes long, e.g. 3h 45m 34s?

Reply:
It doesn't look bad yet.

<img width="305" alt="image" src="https://github.com/sematic-ai/sematic/assets/133257643/f0a9557a-780d-4e81-982d-c302c9df4772">

If we want, we can keep only the two most significant parts if it's too long, and add a tooltip to show the full text. Maybe in the future. But please let me know if you need this addressed immediately.